ethdb/database.go

```diff
@@ -221,6 +222,13 @@ func (db *LDBDatabase) meter(refresh time.Duration) {
 			db.log.Error("Failed to read database stats", "err", err)
 			return
 		}
+		iostats, err := db.db.GetProperty("leveldb.iostats")
+		if err != nil {
+			db.log.Error("Failed to read database iostats", "err", err)
+			return
+		}
+		fmt.Println(iostats)
```

Comment:
Don't print it here, put it into a counter or whatnot like the compactions. Also don;t split the compaction gathering code in half with your code.

Reply:
Oh okay, I thought it should be temporary.